graphql_compiler/schema_transformation/merge_schemas.py

```diff
@@ -161,6 +138,89 @@ def _get_basic_schema_ast(query_type):
     return blank_ast
 
 
+def _merge_single_schema(merged_schema_ast, name_to_schema_id, scalars, directives,
+                         current_schema_id, current_ast):
+    """Merge current_ast into merged_schema_ast, update all records accordingly.
```

Comment:
It seems to me that "update all records accordingly" isn't really saying anything.

Reply:
I wanted to express that `name_to_schema_id`, `scalars`, and `directives` are all potentially getting modified to reflect information in the new schema ast. I can just remove that part of the sentence if it's not useful.